contracts/DHTT.sol

```diff
@@ -0,0 +1,249 @@
+  * @param _owner address The address which owns the funds. 
+  * @param _spender address The address which will spend the funds. 
+  * @return A uint256 specifying the amount of tokens still available for the spender. 
+  */ 
+  function allowance(address _owner, address _spender) public constant returns (uint256 remaining) { 
+    return allowed[_owner][_spender]; 
+  } 
+ 
+ /** 
+  * approve should be called when allowed[_spender] == 0. To increment 
+  * allowed value is better to use this function to avoid 2 calls (and wait until 
+  * the first transaction is mined) * From MonolithDAO Token.sol 
+  */ 
+  function increaseApproval (address _spender, uint _addedValue) public returns (bool success) {
+    allowed[msg.sender][_spender] = allowed[msg.sender][_spender].add(_addedValue);
+    emit Approval(msg.sender, _spender, allowed[msg.sender][_spender]); 
+    return true; 
+  }
+ 
+  function decreaseApproval (address _spender, uint _subtractedValue) public returns (bool success) {
+    uint oldValue = allowed[msg.sender][_spender]; 
+    if (_subtractedValue > oldValue) {
+      allowed[msg.sender][_spender] = 0;
+    } else {
+      allowed[msg.sender][_spender] = oldValue.sub(_subtractedValue);
+    }
+    emit Approval(msg.sender, _spender, allowed[msg.sender][_spender]);
+    return true;
+  }
+ 
+  function () public payable {
+    revert();
+  }
+ 
+}
+ 
+/**
+ * @title Ownable
+ * @dev The Ownable contract has an owner address, and provides basic authorization control
+ * functions, this simplifies the implementation of "user permissions".
+ */
+contract Ownable {
+  address public owner;
+
+ 
+  /**
+   * @dev The Ownable constructor sets the original `owner` of the contract to the sender
+   * account.
+   */
+  function Ownable() public {
+    owner = msg.sender;
+  }
+ 
+ 
+  /**
+   * @dev Throws if called by any account other than the owner.
+   */
+  modifier onlyOwner() {
+    require(msg.sender == owner);
+    _;
+  }
+ 
+}
+ 
+/**
+ * @title Mintable token
+ * @dev Simple ERC20 Token example, with mintable token creation
+ * @dev Issue: * https://github.com/OpenZeppelin/zeppelin-solidity/issues/120
+ * Based on code by TokenMarketNet: https://github.com/TokenMarketNet/ico/blob/master/contracts/MintableToken.sol
+ */
+ 
+contract MintableToken is StandardToken, Ownable {
+  
+  event Mint(address indexed to, uint256 amount);
+  
+ function mint(uint256 _amount) onlyOwner public returns (bool) {
+    require(maxSupply >= totalSupply + _amount);
+    totalSupply = totalSupply.add(_amount);
+    balances[owner] = balances[owner].add(_amount);
+    emit Mint(owner, _amount);
+    return true;
+  }
+
+}
+ 
+contract DHTT is MintableToken {
+    
+    string public constant name = "go to https://try.dehedge.com";
+    
+    string public constant symbol = "DHTT";
+    
+    uint32 public constant decimals = 18;
+    
+    function DHTT() public {        
+        totalSupply = 10000000000000000000000000000000000;
```

Comment:
Of course not so much. We need at most 3 mln tokens.
Also we don't need any decimals at all.